data_driven_acquisition/templates/new.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'base_with_sidebar.html' %}
```

Comment:
Added in #22 to allow the sidebar to show for new acquisitions (required for in-page navigation).